docs/languages/Csharp.md

```diff
@@ -11,9 +12,13 @@ There are special use-cases that each language supports; this document pertains
 <!-- tocstop -->
 
 ## Generate serializer and deserializer functionality
-TODO
+
+To generate the data models based on JSON Schema document and validate the serialized data against the schema, use the preset `CSHARP_JSON_SERIALIZER_PRESET`
```

Comment:
```suggestion
Sometimes you want to serialize the data models into JSON, in order to do that use the preset `CSHARP_JSON_SERIALIZER_PRESET`
```
It is not only related to JSON schema, even though you are correct that instantiated data models should always validate accurately against the schema 🙂 If the models are created from a schema of course, but this does not necessarily have to be the case.

Reply:
Right, I kind of overlooked this. Sorry !